src/Pages/SignIn/SignIn.js

```diff
@@ -0,0 +1,226 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+import SideNav from "../../Components/Nav/SideNav/SideNav.js";
+import Footer from "../../Components/Footer/Footer.js";
+import API_URL from "../../Config.js";
+import "./SignIn.scss";
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      isLoginBtnHovered: false,
+      isSignUpBtnHovered: false,
+      userId: "",
+      userPw: "",
+      isSubmit: false,
+      isValidate: false,
+    };
+  }
+
+  handleUserInfo = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { userId, userPw } = this.state;
+    fetch(`${API_URL}/signin`, {
+      method: "POST",
+      body: JSON.stringify({
+        userid: userId,
+        password: userPw,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        localStorage.setItem("Authorization", res.access_token);
+        if (res.access_token) {
+          this.props.history.push("/main");
+        } else {
+          this.setState({
+            isSubmit: true,
+            isValidate: false,
+          });
+        }
+      });
+  };
+
+  // 수정중
+  // hadleKakaoLogin = () => {
+  //   fetch("http://10.58.6.110:8000/user/signin/kakao")
+  //     .then((res) => res.json())
+  //     .then((res) => {
+  //       localStorage.setItem("Kakao_Token", res.access_token);
+  //       if (res.access_token) {
+  //         this.props.history.push("/main");
+  //         window.location.reload();
+  //       }
+  //     });
+  // };
+
+  render() {
+    const {
+      isLoginBtnHovered,
+      isSignUpBtnHovered,
+      userId,
+      userPw,
+      isSubmit,
+      isValidate,
+    } = this.state;
+
+    return (
+      <>
+        <SideNav />
+        <div className="SignIn">
+          <div className="singinBox">
+            <div className="pageTitArea">
+              <img
+                alt=""
+                src="https://image.drjart.com/front/ko/images/util/icon_loginJoin.gif"
+              />
+              <h2>로그인</h2>
+            </div>
+            <div className="loginCon">

```

Comment:
```suggestion
            <div className="loginContainer">
```
이 경우에는 container라고 끝까지 적어주세요. control의 con인지 contain의 con인지 명확하지 않습니다.

Reply:
수정 완료